### PR TITLE
feat: Add script for reset files references

### PR DESCRIPTION
**Pull request by Gara64**
This removes all the `referenced_by` contained into the files docs.

### CODE REVIEW

File: scripts/resetFilesReferencedBy.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const mkAPI = require('./api')
+
+const DOCTYPE_FILES = 'io.cozy.files'
+
+let client
```
Comment (ptbrowne) [inline scripts/resetFilesReferencedBy.js]: Nit: The client can be declared inside the `run` function.

File: scripts/resetFilesReferencedBy.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          await client.fetchJSON(
+            'DELETE',
+            `/files/${id}/relationships/referenced_by`,
+            ref
+          )
+        }
+      }
+    }
+  }
+}
```
Comment (ptbrowne) [inline scripts/resetFilesReferencedBy.js]: Seems good :) Can you add a test please ?

Comment (Gara64) [reply scripts/resetFilesReferencedBy.js]: Just did!

**Pull request merged**